kyuubi-server/src/main/scala/org/apache/kyuubi/operation/ExecuteStatement.scala

```diff
@@ -59,6 +59,7 @@ class ExecuteStatement(
 
   private def executeStatement(): Unit = {
     try {
+      sendCredentialsIfNeeded()
```

Comment:
what happens if call other operations ? e.g. GetTables

Reply:
these operations also might meet token expiration issue.

Reply:
should we do it in KyuubiOperation::beforeRun?
For other operations, most of them rely on hive token.
cc @yaooqinn

Reply:
seems hive token lift time is more long than hdfs delegation token, so, it is not very critical for us.